great_expectations/data_context/types/base.py

```diff
@@ -1194,7 +1191,7 @@ def validate_schema(self, data, **kwargs) -> None:
                 validation_error=ValidationError(message="config version too high"),
             )
 
-        if data["config_version"] < CURRENT_GX_CONFIG_VERSION and (
+        if data["config_version"] < FIRST_GX_CONFIG_VERSION_WITH_CHECKPOINT_STORE and (
```

Comment:
I think this check can just go away, right? We would have already errored on line 1179